vector/src/main/java/im/vector/app/features/home/room/list/RoomSummaryItem.kt

```diff
@@ -224,14 +211,10 @@ abstract class RoomSummaryItem : VectorEpoxyModel<RoomSummaryItem.Holder>() {
         // val roomAvatarPublicDecorationImageView by bind<ImageView>(R.id.roomAvatarPublicDecorationImageView)
         val roomFailSendingImageView by bind<ImageView>(R.id.roomFailSendingImageView)
         val roomAvatarPresenceImageView by bind<PresenceStateImageView>(R.id.roomAvatarPresenceImageView)
-<<<<<<< HEAD
-        val rootView by bind<ViewGroup>(R.id.itemRoomLayout)
+        val rootView by bind<ConstraintLayout>(R.id.itemRoomLayout)
```

Comment:
Did you verify this is a constraintLayout in tchap layout?

Reply:
it seems so otherwise there should be a crash on the homescreen.

Reply:
yes it is